src/app/globals.css

```diff
@@ -1,3 +1,23 @@
 @tailwind base;
 @tailwind components;
+
+/* Hide scrollbar for Chrome, Safari and Opera */
+.no-scrollbar::-webkit-scrollbar {
+    display: none;
+}
+
+/* Hide scrollbar for IE, Edge and Firefox */
+.no-scrollbar {
+    -ms-overflow-style: none; /* IE and Edge */
+    scrollbar-width: none; /* Firefox */
+}
+
+html {
+    margin-right: calc(100% - 100vw);
+}
+
+body {
+    margin-right: calc(100vw - 100%);
+}
```

Comment:
What is going on? I suppose it is somehow related to the scrollbar, but is it worth it? I don't see any reason to hide the scrollbar as it makes UX worse

Reply:
The initial idea was to get rid of the layout shift when modal is opened and scroll is hidden. Though it might not be a good idea to keep those lines of code